docs/index.md

```diff
@@ -92,11 +147,12 @@ Use Office Scripts in Excel to automate your common tasks. Explore the following
                 <div class="card">
                     <div class="cardImageOuter">
                         <div class="cardImage">
-                            <a href="resources/samples/excel-samples.md" target="_blank"><img src="images/index-landing-page/i_code-samples.svg" alt="Sample scripts" /></a>
+                            <a href="develop/scripting-fundamentals.md" target="_blank"><img src="images/index-landing-page/i_code-blocks.svg" alt="Script fundamentals" /></a>
                         </div>
                     </div>
                     <div class="cardText">
-                        <a href="resources/samples/excel-samples.md" target="_blank"><h3>Sample scripts</h3></a>
+                        <h3>Script fundamentals</h3>
+                        <p><a href="develop/scripting-fundamentals.md">Learn the basics of making Office Scripts in Excel.</a></p>
```

Comment:
Suggestion: Remove description to be consistent with other cards in this section.